components/bpmn/org.wso2.carbon.bpmn.ui/src/main/resources/web/bpmn/instance_list_view.jsp

```diff
@@ -395,7 +397,8 @@
                                         out.print("Not Available");
                                     }%></td>
                             <td><a href=<%="process_list_view.jsp?operation=processDef&processID=" + bpmnInstance.getProcessId()%>><%=bpmnInstance.getProcessId()%></a></td>
-                            <td><%=bpmnInstance.getStartTime().toString()%></td>
+                            <% DateFormat dateOutputFormatter = new SimpleDateFormat("MM/dd/yyyy"); %>
```

Comment:
we cannot define static final variables in JSP. what we can do here is, define it in a Class and reference that class from JSPs. Rather than creating a new java class to hold only this (to keep date format), i think it should be better to keep in the jsp file.